svm/src/transaction_processor.rs

```diff
@@ -824,56 +820,52 @@ impl<FG: ForkGraph> TransactionBatchProcessor<FG> {
         &self,
         callbacks: &CB,
         pubkey: &Pubkey,
-        environments: &ProgramRuntimeEnvironments,
-    ) -> ProgramAccountLoadResult {
-        let program_account = match callbacks.get_account_shared_data(pubkey) {
-            None => return ProgramAccountLoadResult::AccountNotFound,
-            Some(account) => account,
-        };
+    ) -> Option<ProgramAccountLoadResult> {
+        let program_account = callbacks.get_account_shared_data(pubkey)?;
 
         debug_assert!(solana_bpf_loader_program::check_loader_id(
```

Comment:
From a standalone SVM API perspective, I'm undecided on whether it makes sense to
keep this or just return None